runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -27,16 +27,15 @@
 
 import com.google.common.eventbus.Subscribe;
 import com.google.inject.Provides;
+
```

Comment:
These new lines shouldn't be here https://github.com/runelite/runelite/wiki/Code-Conventions#imports
As well as wildcard imports and other import issues.